### PR TITLE
[DP-468] Retrieve Invoice endpoint

### PR DESCRIPTION
**Pull request by pkopac**
* adding retrieve action
* covered with basic unit test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ import chartmogul
 chartmogul.Invoice.create(config, uuid='cus_5915ee5a-babd-406b-b8ce-d207133fb4cb', data={})
 chartmogul.Invoice.all(config, uuid='cus_5915ee5a-babd-406b-b8ce-d207133fb4cb', page=2, per_page=10)
 chartmogul.Invoice.all(config, customer_uuid='cus_f466e33d-ff2b-4a11-8f85-417eb02157a7', external_id='INV0001')
+chartmogul.Invoice.retrieve(config, uuid='inv_123')
```
Comment (jasonl) [inline README.md]: If you're using a UUID here, then display a valid UUID, or it will confuse people.

File: test/api/test_invoice.py [hunk 2]
```diff
@@ -204,6 +204,55 @@
 }
 """
 
+retrieveInvoiceExample = """
+{
+  "uuid": "inv_123",
```
Comment (jasonl) [inline test/api/test_invoice.py]: Again, I'm somewhat surprised that this doesn't cause errors, I guess we must not be actually parsing the UUID from a string anywhere.

Comment (pkopac) [inline test/api/test_invoice.py]: Well, the UUID doesn't parse anywhere in this library & this is just a mocked test.
It's testing that the overall method works and it's parsable into a python object.
Also, we don't validate UUIDs in client libraries, that is done on server side. Here, it's just a string.
I don't want to duplicate unnecessarily.

**Comment by pkopac** [comment]
@jasonl OK now? :)

**Pull request merged**